src/main/java/org/ohdsi/vocabulary/Concept.java

```diff
@@ -0,0 +1,48 @@
+package org.ohdsi.vocabulary;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+
+import java.util.Date;
+import java.util.Objects;
+
+@JsonInclude()
+@JsonIgnoreProperties(ignoreUnknown = true)
+@JsonPropertyOrder({
+        "CONCEPT_ID", "CONCEPT_NAME", "STANDARD_CONCEPT",
+        "STANDARD_CONCEPT_CAPTION", "INVALID_REASON", "INVALID_REASON_CAPTION",
+        "CONCEPT_CODE", "DOMAIN_ID", "VOCABULARY_ID", "CONCEPT_CLASS_ID",
+        "VALID_START_DATE", "VALID_END_DATE"
+})
+public class Concept extends org.ohdsi.circe.vocabulary.Concept {
```

Comment:
This is something that feels messy now:  That the StandardAnalysisAPI is pulling a base class from another library (circe) but then extending it here in SAA to add ValidStartDate/ValidEndDate.

I don't think we need to take action now, but ideally what we should have in the future is that we can define standard objects in SAA (such as Concept) and haave circe-be depend back on SAA to get the 'standard' classes (along with defining it's own cohort specific classes). Eventually, it would be nice to have the circe classes put into SAA (but that is much much further down the road).